src/main/webapp/enterClass.html

```diff
@@ -3,17 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <title>Enter Class</title>
-    <link rel="stylesheet" href="style.css">
```

Comment:
Did you mean to remove the style, if so that's fine just checking

Reply:
I'm going to add a different style sheet in a later PR.